webapp/src/main/java/com/box/l10n/mojito/entity/ThirdPartyTextUnit.java

```diff
@@ -0,0 +1,48 @@
+package com.box.l10n.mojito.entity;
+
+import javax.persistence.*;
+
+/**
+ * Entity used to store the mapping between third party text unit ids and mojito text units.
+ *
+ * @author chidinmae
+ */
+@Entity
+@Table(name = "third_party_text_unit",
+       indexes = {@Index(name = "UK__TP_TEXT_UNIT", columnList = "third_party_text_unit_id", unique = true)})
+public class ThirdPartyTextUnit extends AuditableEntity {
+
+    @Column(name = "third_party_text_unit_id")
+    private String thirdPartyTextUnitId;
+
+    @Column(name = "mapping_key")
+    private String mappingKey;
+
+    @JoinColumn(name = "tm_text_unit_id", foreignKey = @ForeignKey(name = "FK__THIRD_PARTY_TRANSLATION_TEXT_UNIT__TM_TEXT_UNIT__ID"))
```

Comment:
ok could you add that naming convention somewhere I can reference?

Reply:
that's knowledge we share during code reviews

Reply:
also since this PR doesn't contain any "service" logic, it is better to have only the Smartling client code here and move this entity in the PR for the service where it will be used

Reply:
sure thing, will remove